From: KevinDuan <duankaiwen@kylinos.cn>
Date: Tue, 24 Sep 2024 07:08:37 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 100.0% (118 of 118 strings)

Translation: openkylin-nile-new/kylin-calendar
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-calendar/ug/
---
 translation/calendar_kk.ts | 20 ++++++++++----------
 translation/calendar_ky.ts | 18 +++++++++---------
 translation/calendar_ug.ts | 18 +++++++++---------
 3 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/translation/calendar_kk.ts b/translation/calendar_kk.ts
index dff0530..7d22799 100644
--- a/translation/calendar_kk.ts
+++ b/translation/calendar_kk.ts
@@ -30,7 +30,7 @@
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="217"/>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="249"/>
         <source>Schedule Begin</source>
-        <translation>كۇنتارتىبى جۇز بەرگەن ۋاقت</translation>
+        <translation>كۇنتارتىبى جۇز بەرگەن ۋاقىت</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="218"/>
@@ -197,7 +197,7 @@
     <message>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="1016"/>
         <source>不重复</source>
-        <translation>تەكرارلانمىسۇن</translation>
+        <translation>قايتالانبايدى</translation>
     </message>
 </context>
 <context>
@@ -220,7 +220,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="59"/>
         <source>Schedule</source>
-        <translation>كالەندىر</translation>
+        <translation>كالەندٸر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="66"/>
@@ -230,7 +230,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="85"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>ٴسىز نۇ ۋاقىيعانىڭ ٶشٸرۋشٸنٸ جەزىملەشتۈرەلەمٴسىز؟</translation>
+        <translation>ٴسىز نۇ ۋاقىيعانىڭ ٶشٸرۋشٸنٸ جەزملەشتۈرەلەمٴسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="90"/>
@@ -255,7 +255,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="121"/>
         <source>No</source>
-        <translation>كۇشىنەن قالتۇرۇش</translation>
+        <translation>كۇشىنەن قالدىرۋ</translation>
     </message>
 </context>
 <context>
@@ -301,7 +301,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="197"/>
         <source>April</source>
-        <translation>اپٸرەل</translation>
+        <translation>اپرەل</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="200"/>
@@ -336,7 +336,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="218"/>
         <source>November</source>
-        <translation>نويابېر</translation>
+        <translation>نويابىر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="221"/>
@@ -349,7 +349,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="344"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>ٴسىز نۇ ۋاقىيعانىڭ ٶشٸرۋشٸنٸ جەزىملەشتۈرەلەمٴسىز؟</translation>
+        <translation>ٴسىز نۇ ۋاقىيعانىڭ ٶشٸرۋشٸنٸ جەزملەشتۈرەلەمٴسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="502"/>
@@ -479,7 +479,7 @@
     </message>
     <message>
         <source>Friday</source>
-        <translation>اپتانىڭ بەسى</translation>
+        <translation>جۇما</translation>
     </message>
     <message>
         <source>Saturday</source>
@@ -506,7 +506,7 @@
     <message>
         <location filename="../lunarcalendarwidget/notificationsadaptor.cpp" line="103"/>
         <source>Schedule Begin</source>
-        <translation>كۇنتارتىبى جۇز بەرگەن ۋاقت</translation>
+        <translation>كۇنتارتىبى جۇز بەرگەن ۋاقىت</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/notificationsadaptor.cpp" line="113"/>
diff --git a/translation/calendar_ky.ts b/translation/calendar_ky.ts
index 9023106..4f19c05 100644
--- a/translation/calendar_ky.ts
+++ b/translation/calendar_ky.ts
@@ -193,7 +193,7 @@
     <message>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="1016"/>
         <source>不重复</source>
-        <translation>تەكرارلانمىسۇن</translation>
+        <translation>قايتىلانمايدۇ</translation>
     </message>
 </context>
 <context>
@@ -216,7 +216,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="59"/>
         <source>Schedule</source>
-        <translation>كالىندار</translation>
+        <translation>كالەندار</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="66"/>
@@ -226,7 +226,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="85"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>سىز بۇل وقۇيانى ۅچۉرۉۉنۉ تۇراقتاندىرا الاسىزبى؟</translation>
+        <translation>سىز بۇل وقۇيانى ۅچۉرۉۉنۉ جەزملەشتۈرەلەمسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="90"/>
@@ -251,7 +251,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="121"/>
         <source>No</source>
-        <translation>ارعادان  قالتۇرۇش</translation>
+        <translation>ارعادان  قالتىرىش</translation>
     </message>
 </context>
 <context>
@@ -297,7 +297,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="197"/>
         <source>April</source>
-        <translation>ئاپرىل</translation>
+        <translation>اپرىل</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="200"/>
@@ -322,7 +322,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="212"/>
         <source>September</source>
-        <translation>سىنتابىر</translation>
+        <translation>سەنتابىر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="215"/>
@@ -332,7 +332,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="218"/>
         <source>November</source>
-        <translation>نويابېر</translation>
+        <translation>نويابىر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="221"/>
@@ -345,7 +345,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="344"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>سىز بۇل وقۇيانى ۅچۉرۉۉنۉ تۇراقتاندىرا الاسىزبى؟</translation>
+        <translation>سىز بۇل وقۇيانى ۅچۉرۉۉنۉ جەزملەشتۈرەلەمسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="502"/>
@@ -467,7 +467,7 @@
     </message>
     <message>
         <source>Friday</source>
-        <translation>اپتانىن بەشى</translation>
+        <translation>جۇما</translation>
     </message>
     <message>
         <source>Saturday</source>
diff --git a/translation/calendar_ug.ts b/translation/calendar_ug.ts
index 9051dc9..3b41038 100644
--- a/translation/calendar_ug.ts
+++ b/translation/calendar_ug.ts
@@ -30,7 +30,7 @@
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="217"/>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="249"/>
         <source>Schedule Begin</source>
-        <translation>كۈنتەرتىپ يۈز بەرگەن ۋاقت</translation>
+        <translation>كۈنتەرتىپ يۈز بەرگەن ۋاقىت</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/schedulewidget.cpp" line="218"/>
@@ -216,7 +216,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="59"/>
         <source>Schedule</source>
-        <translation>كالىندار</translation>
+        <translation>كالېندار</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="66"/>
@@ -226,7 +226,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="85"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>سىز بۇ ۋەقەنى ئۆچۈرۈشنى جەزىملەشتۈرەلەمسىز؟</translation>
+        <translation>سىز بۇ ۋەقەنى ئۆچۈرۈشنى جەزملەشتۈرەلەمسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="90"/>
@@ -251,7 +251,7 @@
     <message>
         <location filename="../lunarcalendarwidget/custommessagebox.cpp" line="121"/>
         <source>No</source>
-        <translation>ئەمەلدىن قالتۇرۇش</translation>
+        <translation>ئەمەلدىن قالدۇرۇش</translation>
     </message>
 </context>
 <context>
@@ -297,7 +297,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="197"/>
         <source>April</source>
-        <translation>ئاپرىل</translation>
+        <translation>ئاپرېل</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="200"/>
@@ -322,7 +322,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="212"/>
         <source>September</source>
-        <translation>سىنتەبىر</translation>
+        <translation>سېنتەبىر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="215"/>
@@ -332,7 +332,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="218"/>
         <source>November</source>
-        <translation>نويابېر</translation>
+        <translation>نويابىر</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarmonthitem.cpp" line="221"/>
@@ -345,7 +345,7 @@
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="344"/>
         <source>Are you sure you want to delete this event?</source>
-        <translation>سىز بۇ ۋەقەنى ئۆچۈرۈشنى جەزىملەشتۈرەلەمسىز؟</translation>
+        <translation>سىز بۇ ۋەقەنى ئۆچۈرۈشنى جەزملەشتۈرەلەمسىز؟</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/lunarcalendarwidget.cpp" line="502"/>
@@ -494,7 +494,7 @@
     <message>
         <location filename="../lunarcalendarwidget/notificationsadaptor.cpp" line="103"/>
         <source>Schedule Begin</source>
-        <translation>كۈنتەرتىپ يۈز بەرگەن ۋاقت</translation>
+        <translation>كۈنتەرتىپ يۈز بەرگەن ۋاقىت</translation>
     </message>
     <message>
         <location filename="../lunarcalendarwidget/notificationsadaptor.cpp" line="113"/>
